src/state.rs

```diff
@@ -41,11 +41,12 @@ pub mod policy;
 pub mod service;
 pub mod workload;
 
-#[derive(Debug, Hash, Eq, PartialEq, Clone, serde::Serialize)]
```

Comment:
The Hash trait cannot been derived for `Option<Arc<Service>>`, and I think this trait is not been used by Upstream, so I removed it. @howardjohn